--- a/src/commonMain/kotlin/androidx/wear/compose/material3/AnimatedCornerShape.kt
+++ b/src/commonMain/kotlin/androidx/wear/compose/material3/AnimatedCornerShape.kt
@@ -37,8 +37,8 @@
 import androidx.compose.ui.graphics.Matrix
 import androidx.compose.ui.graphics.Outline
 import androidx.compose.ui.graphics.Shape
-import androidx.compose.ui.graphics.asComposePath
 import androidx.compose.ui.unit.Density
+import ee.schimke.wearcmp.port.toComposePath
 import androidx.compose.ui.unit.LayoutDirection
 import androidx.compose.ui.unit.LayoutDirection.Ltr
 import androidx.compose.ui.util.lerp
@@ -152,7 +152,7 @@
         }
 
         val path =
-            morph.toPath(progress()).asComposePath().apply {
+            morph.toComposePath(progress()).apply {
                 transform(Matrix().apply { scale(size.width, size.height) })
             }
 
