--- a/src/commonMain/kotlin/androidx/wear/compose/material3/onehandedgesture/OneHandedGestureClickIndicator.kt
+++ b/src/commonMain/kotlin/androidx/wear/compose/material3/onehandedgesture/OneHandedGestureClickIndicator.kt
@@ -19,7 +19,6 @@
 package androidx.wear.compose.material3.onehandedgesture
 
 import androidx.compose.animation.core.Animatable
-import androidx.compose.animation.graphics.res.rememberAnimatedVectorPainter
 import androidx.compose.foundation.layout.Box
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.Stable
